website/README.md

```diff
@@ -14,13 +14,13 @@ You will need the following things properly installed on your computer.
 
 ## Installation
 
-* `git clone <repository-url>` this repository
+* `git clone https://github.com/hashicorp/design-system.git` this repository
 * `cd website`
-* `npm install`
+* `yarn`
 
 ## Running / Development
 
-* `ember serve`
+* `yarn start`
```

Comment:
should we add (and explain) the `start:prember` yarn command that one could use?

Reply:
👍  added another bullet point with that